### PR TITLE
Add flag to install dependency only

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?

* Add a new flag to install dependency only by `install.sh` 
* Update install.sh 
* Use `IfNotPresent` instead of `Always`

### CODE REVIEW

File: helm/chaos-mesh/values.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ controllerManager:
   replicaCount: 1
 
   image: pingcap/chaos-mesh:latest
-  imagePullPolicy: Always
+  imagePullPolicy: IfNotPresent
```
Comment (Yisaer) [inline helm/chaos-mesh/values.yaml:24]: Why change to `IfNotPresent`

Comment (cwen0) [reply helm/chaos-mesh/values.yaml:24]: I think `always` is not necessary. Sometimes if the pod was restarted and we change some flags of it, which may cause the pod crashing.

Comment (Gallardot) [reply helm/chaos-mesh/values.yaml:24]: I think we should keep it set to `always`. Because our image is `latest`.  Unless we specify a particular version.

Comment (cwen0) [reply helm/chaos-mesh/values.yaml:24]: Got it

**Pull request merged**